topiary/src/atom_collection.rs

```diff
@@ -773,63 +773,61 @@ fn detect_multi_line_nodes(node: &Node) -> HashSet<usize> {
 }
 
 fn detect_blank_lines_before(node: &Node) -> HashSet<usize> {
-    detect_line_breaks_inner(node, 2, None, None).0
+    detect_line_breaks_inner(node, 2).0
```

Comment:
This will be even nicer if the output from `detect_line_breaks_inner` is a struct rather than a tuple.